src/highlevel_api.jl

```diff
@@ -295,3 +295,20 @@ function _bytes2string(x::Integer)
 end
 
 has_object_of_type(t) = any(obj -> obj.type_ == t, gettopology())
+
+"""
+Shows a graphical visualization of the system topology.
+The quality of the result might depend on the used terminal and might vary between machines and operating systems.
+
+**Note:** The specific visualization may change between minor versions.
+"""
+function topology_graphical()
+    if isdefined(Hwloc_jll, :lstopo_no_graphics)
+        Hwloc_jll.lstopo_no_graphics() do lstopo
+            run(`$lstopo --no-io --no-legend --of txt`)
+        end
```

Comment:
This is slightly simpler (and [better](https://docs.binarybuilder.org/stable/jll/#ExecutableProduct))
```suggestion
        run(`$(lstopo_no_graphics()) --no-io --no-legend --of txt`)
```